lib/benchee/statistics.ex

```diff
@@ -153,8 +153,8 @@ defmodule Benchee.Statistics do
   def statistics(suite = %Suite{scenarios: scenarios}) do
     scenarios_with_statistics =
       Parallel.map(scenarios, fn scenario ->
-        run_time_stats = job_statistics(scenario.run_times)
-        memory_stats = job_statistics(scenario.memory_usages)
+        run_time_stats = job_statistics(scenario.run_times, :run_time)
+        memory_stats = job_statistics(scenario.memory_usages, :memory)
```

Comment:
I thought more about it in terms of leaving `job_statistics` alone and extracting the calculation of ips stuff from there. Then we'd just add the IPs stuff in here for the runtime_stats and just not touch it at all for memory_stats.

That way we'd avoid the ifs/conditionals, could safely put the values on/leave them at nil for memory stats. Also for the more long term plan of extracting statistics from benchee that might be nicer because IPS is very benchee specific while I might still want to have an equivalent of job_statistics in the statistics library.

What do you think? :)

as clarification something along the lines of:

```elixir
run_time_stats = scenario.run_times |> job_statistics() |> add_ips()
memory_stats = job_statistics(scenario.memory_usages)
```